src/app/Http/Requests/FinancialAccounts/ShowOrExportOperationsRequest.php

```diff
@@ -2,9 +2,9 @@
 
 namespace App\Http\Requests\FinancialAccounts;
 
-use Illuminate\Foundation\Http\FormRequest;
+use App\Http\Requests\Base\DateRequest;
 
-class ShowOrExportOperationsRequest extends FormRequest
+class ShowOrExportOperationsRequest extends DateRequest
```

Comment:
zvážil by som ešte, či je táto trieda potrebná, keď oproti rodičovskej nemá žiadnu funkcionalitu navyše (hoci chápem sémanticky význam jej existencie)

Reply:
predpokladám, že myslíš ShowOrExportOperationsRequest...áno, DateRequest má zmysel ako všeobecná (až abstraktná) trieda a táto sémanticky ako jej špecializácia, tak som to myslel, aj keď uznávam, že žiadnu novú funkcionalitu neprináša...neviem, oboma spôsobmi by mi to sedelo, použitie ShowOrExportOperationsRequest je akurát pre mňa také čitateľnejšie...zatiaľ to asi nechám, je totiž možné, že po zmene organizácie kontrolerov, o ktorej som hovoril, by sa dala autorizácia všade presunúť do týchto requestov